frontend/src/components/widgets/AssigneesList.jsx

```diff
@@ -1,18 +1,22 @@
 import React from "react";
-// import { Typography } from "@material-ui/core";
-import { Table, TableBody, TableRow, TableCell } from "@material-ui/core";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import UserAvatarLink from "../widgets/UserAvatarLink";
 
-export default ({ userNames, userIds }) => {
+export default function AssigneesList({ userNames, userIds }) {
   return (
     <Table size="small">
       <TableBody>
         {userNames.map((name) => (
-          <TableRow>
-            <TableCell padding="none">{name}</TableCell>
+          <TableRow key={userIds}>
```

Comment:
userIds should be an array of Ids, not just one id.  That is why it is called `userIds` and not `userId`

eg input would be:

```
{
userNames: ["you@umuzi.org", "me@umuzi.org"],
userIds:        [3,                            2],
}
````

Rather:

```
userNames.map((name,index)=>{
    const userId = userIds[index];
   return (<TableRow key={userId}> ..... </TableRow>)
})
```